src/main/flight/position.c

```diff
@@ -203,7 +199,7 @@ void calculateEstimatedAltitude(void)
     previousZeroedAltitudeCm = zeroedAltitudeCm;
 
     // assess if altitude is low here, only when we get new data, rather than in pid loop etc
-    altitudeIsLow = zeroedAltitudeCm < 100.0f * positionConfig()->landing_altitude_m;
+    altitudeIsLow = zeroedAltitudeCm < 100.0f * positionControlConfig()->landing_altitude_m;
```

Comment:
This is a circular dependency (altitude.h depends on altitude_control.h and vice versa). `isAltitudeLow()` seems to be special and probably belongs somewhere else (for example in position control), with another name `isLandingAltitudeReached()`
```c
bool isLandingAltitudeReached()
{
    return getAltitudeCm() < 100.0f * positionControlConfig()->landing_altitude_m;
}
```

Reply:
Yes, makes more sense within `position_control.c`.  Implemented using `isBelowLandingAltitude` as the label. Done.